client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -9,6 +9,9 @@ uniform vec3 eyePosition;
 // The cameraOffset is the current center of the visible world.
 uniform vec3 cameraOffset;
 uniform float animationTimer;
+
+uniform float ambientLight;
+uniform vec3 ambientColor;
```

Comment:
A single `ambientColor` uniform should suffice. `ambientLight` can be multiplied by `ambientColor` outside the shader.